src/test/functional/cypress/integration/api/admin/empStatus_API.spec.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+const empStatusAPI = '/api/v2/admin/employment-statuses';
+
+describe('Admin - Employment Status API ', function () {
+  before(function () {
+    cy.fixture('user').then(({admin}) => {
+      cy.apiLogin(admin);
+    });
+  });
+
+  beforeEach(function () {
+    cy.fixture('chars').as('strings');
+    Cypress.Cookies.preserveOnce('orangehrm', '_orangehrm');
```

Comment:
If we login before each, we might not have to use `preserveOnce()` ?

```
beforeEach(function () {
    cy.fixture('chars').as('strings');
    cy.fixture('user').then(({admin}) => {
      cy.apiLogin(admin);
    });
});
```

Reply:
tried to increase execution speed. but the tester should have a better understanding to use this I guess